MoreCyclopsUpgrades/Modules/Recharging/Nuclear/NuclearChargingManager.cs

```diff
@@ -31,6 +31,15 @@ private static InventoryItem SpawnDepletedRod()
             Pickupable pickupable = gameObject.GetComponent<Pickupable>().Pickup(false);
             return new InventoryItem(pickupable);
         }
+
+        private static InventoryItem SpawnDepletedModule()
```

Comment:
With this method here, `SpawnDepletedRod()` is no longer needed and can be deleted.

Reply:
Removed